### PR TITLE
improving short descriptions for cli commands

### PR DESCRIPTION
**Pull request by frodopwns**
Improving the short descriptions for top level kops commands.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1590)

### CODE REVIEW

**Comment by frodopwns** [comment]
@kris-nova

**Comment by kris-nova** [comment]
@k8s-bot ok to test

File: cmd/kops/completion.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func NewCmdCompletion(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "completion",
-		Short:   "Output shell completion code for the given shell (bash)",
+		Short:   "Output shell completion code for the given shell (bash).",
 		Long:    longDescription,
```
Comment (chrislovecnm) [inline cmd/kops/completion.go]: This is what is it does, but does not help me understand as a n00b, how it helps me

File: cmd/kops/delete.go [hunk 2]
```diff
@@ -30,7 +31,7 @@ func NewCmdDelete(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:        "delete",
-		Short:      "delete clusters",
+		Short:      "Delete clusters and other resources.",
```
Comment (chrislovecnm) [inline cmd/kops/delete.go]: What other resources? Resources associated to the cluster? Not sure how to say that short and sweet.

File: cmd/kops/edit.go [hunk 3]
```diff
@@ -17,15 +17,16 @@ limitations under the License.
...
+
+	"github.com/spf13/cobra"
 	"k8s.io/kops/cmd/kops/util"
 )
 
 func NewCmdEdit(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "edit",
-		Short: "edit items",
+		Short: "Edit clusters and other resrouces.",
```
Comment (chrislovecnm) [inline cmd/kops/edit.go]: Typo ;) 

resources

Comment (chrislovecnm) [inline cmd/kops/edit.go]: Edit cluster and associated resources or supporting resources??

File: cmd/kops/get.go [hunk 4]
```diff
@@ -37,7 +37,7 @@ var getCmd = GetCmd{
 	cobraCommand: &cobra.Command{
 		Use:        "get",
 		SuggestFor: []string{"list"},
-		Short:      "list or get objects",
+		Short:      "List all instances of a resource.",
```
Comment (chrislovecnm) [inline cmd/kops/get.go]: So right now we can get a ig or a cluster ... how do we say that?

File: cmd/kops/import.go [hunk 5]
```diff
@@ -23,7 +23,7 @@ import (
 // importCmd represents the import command
 var importCmd = &cobra.Command{
 	Use:   "import",
-	Short: "import clusters",
+	Short: "Import existing resources into the state store.",
 	Long:  `import clusters`,
```
Comment (chrislovecnm) [inline cmd/kops/import.go]: This is import a cluster created by kube-up.sh

File: cmd/kops/replace.go [hunk 6]
```diff
@@ -40,7 +40,7 @@ func NewCmdReplace(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "replace -f FILENAME",
-		Short: "Replace a resource by filename or stdin",
+		Short: "Replace a resource by filename or stdin.",
```
Comment (chrislovecnm) [inline cmd/kops/replace.go]: Replace a cluster... I need some word smithing and help on this one. Wth does this func do ;)

File: cmd/kops/create.go [hunk 7]
```diff
@@ -44,7 +45,7 @@ func NewCmdCreate(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "create -f FILENAME",
-		Short: "Create a resource by filename or stdin",
+		Short: "Create a resource by filename or stdin.",
```
Comment (chrislovecnm) [inline cmd/kops/create.go]: Again create a k8s cluster

File: cmd/kops/rollingupdate.go [hunk 8]
```diff
@@ -17,15 +17,16 @@ limitations under the License.
...
+	"github.com/spf13/cobra"
 	"k8s.io/kops/cmd/kops/util"
 )
 
 func NewCmdRollingUpdate(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "rolling-update",
-		Short: "rolling update clusters",
+		Short: "Initiate rolling updates on clusters.",
 		Long:  `rolling update clusters`,
```
Comment (chrislovecnm) [inline cmd/kops/rollingupdate.go]: Singular - a cluster

File: cmd/kops/toolbox.go [hunk 9]
```diff
@@ -17,15 +17,16 @@ limitations under the License.
...
+
+	"github.com/spf13/cobra"
 	"k8s.io/kops/cmd/kops/util"
 )
 
 func NewCmdToolbox(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "toolbox",
-		Short: "Misc infrequently used commands",
+		Short: "Misc infrequently used commands.",
```
Comment (chrislovecnm) [inline cmd/kops/toolbox.go]: We need to figure this out. Never used this option. What does it do?

File: cmd/kops/update.go [hunk 10]
```diff
@@ -17,15 +17,16 @@ limitations under the License.
...
+	"github.com/spf13/cobra"
 	"k8s.io/kops/cmd/kops/util"
 )
 
 func NewCmdUpdate(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "update",
-		Short: "update clusters",
+		Short: "Creates or updates cloud resources to match cluster spec.",
 		Long:  `Update clusters`,
```
Comment (chrislovecnm) [inline cmd/kops/update.go]: Updates may mean a create. Not sure if a n00b knows what a cluster spec is.

"Updates a cluster to current spec"

Ideas?

File: cmd/kops/upgrade.go [hunk 11]
```diff
@@ -23,7 +23,7 @@ import (
 // upgradeCmd represents the upgrade command
 var upgradeCmd = &cobra.Command{
 	Use:   "upgrade",
-	Short: "upgrade clusters",
+	Short: "Automates checking for and applying Kubernetes updates.",
```
Comment (chrislovecnm) [inline cmd/kops/upgrade.go]: This is tricky. This upgrades a cluster to match the version in the stable channel. May result in a downgrade as well.

How do we say that in 4 words?

File: cmd/kops/version.go [hunk 12]
```diff
@@ -30,7 +30,7 @@ type VersionCmd struct {
 var versionCmd = VersionCmd{
 	cobraCommand: &cobra.Command{
 		Use:   "version",
-		Short: "Print the client version information",
+		Short: "Print the client version information.",
```
Comment (chrislovecnm) [inline cmd/kops/version.go]: Print the kops version information. Not the client this prints kops.

**Comment by justinsb** [comment]
@frodopwns thanks - nice improvements.  I think I forced a rebase on you though - sorry!

@chrislovecnm let's merge this as is.  I like your suggestions, and you should PR them.  But what @frodopwns has suggested here is already a big improvement, and we should get them in quickly and then we can get more improvements in on top if needed.  And we should avoid rebases :-)

**Comment by kris-nova** [comment]
@frodopwns If we can get a quick rebase, we can get this in.. Thanks for your hard work!

File: cmd/kops/completion.go (hunk 1 above)
Comment (chrislovecnm) [inline cmd/kops/completion.go]: Have some tweaks that I am recommending :)

**Comment by chrislovecnm** [comment]
@kris-nova I have a few comments open, waiting for @frodopwns to respond.

@frodopwns appreciate it :)

File: cmd/kops/completion.go (hunk 1 above)
Comment (kris-nova) [inline cmd/kops/completion.go]: @chrislovecnm,

Im sorry, but this comment is intelligible. What exactly are you trying to communicate here?

**Comment by justinsb** [comment]
Hi @frodopwns would you mind rebasing (there's a conflict on cmd/kops/edit.go, hopefully an easy one) so we can merge?

**Comment by justinsb** [comment]
Thanks @frodopwns  - figured out a little trick!

**Pull request merged**